kudos-webapps/src/main/webapp/vue-app/kudos-overview/components/KudosOverview.vue

```diff
@@ -52,7 +52,40 @@
         </v-col>
       </v-row>
     </widget-wrapper>
-
+    <v-row
+      v-else
```

Comment:
Sorry, I don't get the purpose of modification. Did we added a reusable component `widget-wrapper` to reduce the code or to increase it. Shouldn't we delete the used source code or alter it to reuse the Widget classes only ?